xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Question is, do we care that much about the class creating a copy instance of itself? I get what you're trying to do, but I can see no harm done if some code in CPVRRecording itself creates a copy really.

Re friend class: I consider them a hack, as they slash through all the protection that we put in place. Someone could touch private members that we don't expect to be changed from a friend class, and all those kinds of "nice" things. I can also see no problem if someone wants to allocate an instance of CPVRRecording. So we should avoid friend class statements if we can.